libyuv fails with multiple rules generate yuv.lib error

I am not able to get libyuv to build. I tried the current version in gvsbuild, the latest stable, and latest main version.

I poked around a bit, but I am not that familiar with CMake.

```
(git) Exporting directory C:\gtk-build\build\x64\release\libyuv
Building project libyuv (git/eec8dd37e827a78c3bdbb66da6caad89f4b8c4dd)
(Stripping trailing CRs from patch; use --binary to disable.)
patching file CMakeLists.txt
Hunk #2 succeeded at 76 (offset -6 lines).
-- The C compiler identification is MSVC 19.31.31105.0
-- The CXX compiler identification is MSVC 19.31.31105.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.31.31103/bin/Hostx64/x64/cl.exe - skipped       
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Deprecation Warning at CMakeLists.txt:6 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Found JPEG: C:/gtk-build/gtk/x64/release/lib/jpeg.lib (found version "62") 
CMake Warning (dev) at CMakeLists.txt:46 (if):
  Policy CMP0064 is not set: Support new TEST if() operator.  Run "cmake
  --help-policy CMP0064" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  TEST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
This warning is for project developers.  Use -Wno-dev to suppress it.

Building ver.: 0.0.
Packaging for: amd-64
-- Configuring done
-- Generating done
CMake Error:
  Running

   'C:/gtk-build/tools/ninja-1.10.2/ninja.exe' '-C' 'C:/gtk-build/build/x64/release/libyuv/_gvsbuild-cmake' '-t' 'recompact'

  failed with:

   ninja: error: build.ninja:1071: multiple rules generate yuv.lib [-w dupbuild=err]





CMake Warning:
  Manually-specified variables were not used by the project:

    GTK_DIR


CMake Generate step failed.  Build files cannot be regenerated correctly.
Traceback (most recent call last):
  File "C:\gtk-build\github\gvsbuild\gvsbuild\utils\builder.py", line 552, in build
    if self.__build_one(p):
  File "C:\gtk-build\github\gvsbuild\gvsbuild\utils\builder.py", line 688, in __build_one
    skip_deps = proj.build()
  File "C:\gtk-build\github\gvsbuild\gvsbuild\projects\libyuv.py", line 43, in build
    CmakeProject.build(self, use_ninja=True)
  File "C:\gtk-build\github\gvsbuild\gvsbuild\utils\base_builders.py", line 136, in build
    self.builder.exec_vs(cmd, working_dir=work_dir)
  File "C:\gtk-build\github\gvsbuild\gvsbuild\utils\builder.py", line 974, in exec_vs
    self.__execute(
  File "C:\gtk-build\github\gvsbuild\gvsbuild\utils\builder.py", line 1080, in __execute
    subprocess.check_call(args, cwd=working_dir, env=env, shell=True)
  File "C:\Python310\lib\subprocess.py", line 369, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'cmake -G "Ninja" -DCMAKE_INSTALL_PREFIX="C:\gtk-build\build\x64\release\libyuv-rel" -DGTK_DIR="C:\gtk-build\gtk\x64\release
" -DCMAKE_BUILD_TYPE=RelWithDebInfo -BC:\gtk-build\build\x64\release\libyuv\_gvsbuild-cmake -HC:\gtk-build\build\x64\release\libyuv' returned non-zero exit status 1.
Error: libyuv build failed
```

--- a/gvsbuild/projects/libyuv.py
+++ b/gvsbuild/projects/libyuv.py
@@ -40,7 +40,7 @@ class Libyuv(GitRepo, CmakeProject):
         )
 
     def build(self):
-        CmakeProject.build(self, use_ninja=True)
+        CmakeProject.build(self, use_ninja=False)
 
         self.install_pc_files()
         self.install(r".\LICENSE share\doc\libyuv")

